blog/_posts/2017-08-10-opensource-here-we-come.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Open source, here we come!
+layout: post
+created_at: Monday August 10 2017
+permalink: blog/2017-08-10-opensource-here-we-come
+current: blog
+author: Gemini
+twitter: Team_Gemini_
+categories:
+- student-posts
+---
+
+# Duo of the moment
+
+![image](/img/blog/2017/TeamGemini.jpg)
+<div align="center" class="image-credits"> Team Gemini(Image Courtesy: Shagun Kapur)</div>
+
+[Akarsha](https://github.com/aksh98) and [Ramya](https://github.com/meriki) are not your regular individuals, a dash of crazy, a drop of weird and a couple of undiscovered unusual ingredients makes them up.  They can be found studying in the same college of IIIT Delhi and sharing the same hostel room, adding a whole new meaning to birds of a feather flock together. Their acquaintance started with their mutual interest in coding and fascination for artificial intelligence. Slowly but steadily they started bonding and realized that despite one being from North India and the other from South, one being a die hard Korean fan and the other a Game of Thrones fan, they are eerily similar. Their sun signs (which both are extremely fond of) also matched! While looking for the perfect team name, ‘Gemini’ which literally means ‘The Twins’ made perfect sense. In the short span of two years, they have managed to gather all sorts of memories with each other, reading for tests together, doing assignments together, bickering and laughing the next moment, watching youtube videos, heated discussions on random topics or just not seeing eye to eye with one other. They’ve been through it all and vehemently attest that their bond keeps going strong because of all their experiences together.
+
+# Fun facts about the team
+
+### [__Akarsha:__](https://github.com/aksh98)
+
+- Didn’t manage to grow up beyond the age of 14 :p
+- She is too excited about every new thing that comes her way.
+- She is a die hard Korean fan (finds them cute for some reason) and her long time dream is to visit South Korea. Though I am pretty sure she would love to visit any place as long as she gets to travel.
+- She is a confused soul with a 100 things on her plate.
+- She dreams to be a successful Entrepreneur. 
+- Wants her life to be fun filled and exciting always. 
+
+### [__Ramya:__](https://github.com/meriki)
+
+- Keeps her stuff really clean :P
+- Very lazy at times. Mastered the art of doing more in less
+- Loves fan-fiction, can’t survive without Harry Potter fanfics at least once a week
+- Gets her fat from nowhere(she hardly eats anything.. And if she does, she exercises to burn those calories)
+- Wants to know about anything and everything interesting under the sun and gets very excited every time she manages to do this. Her youtube feed is filled with random explanations about random things
+- Wants her life to be one of adventure and magic
+
+# Why Susi AI?
```

Comment:
Writing about SUSI AI project and then about sci-kit image mentors in introduction of teams looks bit odd. Can you please add the section 'our projects' or your 'What we’ve done so far and some unexpected events' section over here? There you can talk about susi AI project in 2-3 lines, your experience and what is the current situation now. And after that section you can have the same section of introducing the team as it is. You can also have 'future plan' kind of small section at the end, if you want to add that.